tests/e2e/test_simple_registry_workload.py

```diff
@@ -0,0 +1,71 @@
+import os
+import shlex
+from subprocess import Popen, PIPE
+from ocs_ci.framework.testlib import workloads
+import logging
+import shutil
+import pytest
+from ocs_ci.utility.utils import run_cmd
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def svt_project_clone():
+    run_cmd("git clone https://github.com/openshift/svt.git")
+
+
+@pytest.fixture(scope='function')
+def svt_create_venv_setup():
+    run_cmd("virtualenv -p /bin/python2 venv")
+    run_cmd("/bin/sh -c 'source venv/bin/activate && python --version'")
+    run_cmd("/bin/sh -c 'source venv/bin/activate && pip install -r registry_requirement.txt'")
+
+
+@pytest.fixture(scope='function')
+def svt_cluster_loader(clusterload_file="svt/openshift_scalability/config/master-vert.yaml"):
+    KUBECONFIG = os.environ["KUBECONFIG"]
+    """
+    https://github.com/openshift/svt/blob/master/openshift_scalability/README.md
+
+    Arguments:
+    -f : This is the input config file used to define the test.
+    -kubeconfig : kebeconfig path
+    """
+
+    cmd = f"/bin/sh -c 'source venv/bin/activate && python svt/openshift_scalability/cluster-loader.py \
+    -f {clusterload_file} --kubeconfig {KUBECONFIG}'"
+    run_cmd(cmd)
+
+
+def svt_cleanup():
+    try:
+        shutil.rmtree('svt')
+        shutil.rmtree('venv')
+    except BaseException:
+        log.error("Error while cleaning SVT project")
+
+
+@workloads
+class TestRegistryWorkload:
+    def test_registry_workload(self, svt_project_clone, svt_create_venv_setup, svt_cluster_loader, iterations=1):
+        """
+        https://github.com/openshift/svt/blob/master/openshift_performance/ose3_perf/scripts/build_test-README.md
+        The build_test.py scripts is a flexible tool for driving builds in OpenShift.
+        It can execute builds concurrently, sequentially, randomly or in arbitrary combinations.
+
+        Arguments:
+        -z : No-login(oc login)
+        -a : Run all builds in all projects
+        -n : Number of iterations (Do not use more than 10)
+        """
+
+        cmd = f"/bin/sh -c 'source venv/bin/activate && \
+        python svt/openshift_performance/ose3_perf/scripts/build_test.py -z -a -n {iterations}'"
+        log.info(f"Running command {cmd}")
+        build = Popen(shlex.split(cmd), stdout=PIPE, stderr=PIPE)
+        stdout, stderr = build.communicate()
+        out = stderr.split('\n'.encode())[-10].split()[-1]
+        log.info(stderr)
```

Comment:
what would be the value when there are no errors and why do you want to pass an error as info?

Reply:
Build test script logger output is in stderr this will add the build script output in run-ci output